digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -1832,6 +1832,14 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
     }
   }
 
+  get("/getAssetTypes") {
+    AssetTypeInfo.values.filterNot(_.typeId == 99).toList.sortBy(_.typeId).map { asset =>
+      Map("id" -> asset.typeId,
+        "name" -> { if (asset.nameFI.nonEmpty) asset.nameFI else asset.label }
```

Comment:
Why do you need this "if"? You created this "nameFI" and give the names, is there some asset you did not give the name?

Reply:
In case you you want to maintain this implementation 
``` Scala
 "name" -> { if (asset.nameFI.nonEmpty) asset.nameFI else asset.label }
```
You might want to trim before you check for nonEmpty

Reply:
> Why do you need this "if"? You created this "nameFI" and give the names, is there some asset you did not give the name?

But for some reason if no one fill that value or it is empty, I will use the label as a backup attempt to send an empty description to frontend.

Reply:
Ok

Reply:
Added the trim